augur/filter.py

```diff
@@ -1417,6 +1417,10 @@ def run(args):
             if args.output_log:
                 output_log_writer.writerow(filtered_strain)
 
+        # Prevent force-included sequences from being included again during
+        # subsampling.
+        seq_keep = seq_keep - distinct_sequences_to_include
+
         if group_by:
             # If grouping, track the highest priority metadata records or
             # count the number of records per group. First, we need to get
```

Comment:
I might just be a little lost here, but is there a reason why this can't be within the `if group_by:` block?

Reply:
No that's sufficient, and I prefer that scoping! Updated in d97611fb441c390af97deec5170161b3eb3d0a91